cluster/pulumi/cluster/README.md

```diff
@@ -0,0 +1,5 @@
+# cluster (Pulumi project)
+
+Used for setting up fresh GCP clusters for Splice-related deployments.
```

Comment:
it doesn't actually set up a fresh cluster atm though does it? We primarily use it to control node pools on an existing cluster I believe?